code/src/java/pcgen/gui2/facade/CharacterFacadeImpl.java

```diff
@@ -2164,14 +2166,39 @@ void refreshLanguageList()
 				currBonusLangs.add(lang);
 			}
 		}
-		int bonusLangRemain = bonusLangMax - currBonusLangs.size();
+		int bonusLangRemain = 0;
+		if (!theCharacter.getRace().getDisplayName().equals("<none selected>"))
```

Comment:
We generally try and avoid using display name as a method of checking in code. 
Perhaps use `theCharacter.getRace().isUnselected()` ?

Reply:
I agree. I didn't know of another way to check the race, but this is much more robust. Updated